### PR TITLE
TypeScript: type fixes

### PR DESCRIPTION
**Pull request by taphos**

### CODE REVIEW

File: src/scenes/Scene.d.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { Texture } from '../textures/Texture';
 import { WebGLRenderer } from "../renderers/WebGLRenderer";
 import { Camera } from "../cameras/Camera";
 import { RenderTarget } from "../renderers/webgl/WebGLRenderLists";
+import { WebGLRenderTarget } from "../renderers/WebGLRenderTarget";
```
Comment (Mugen87) [inline src/scenes/Scene.d.ts:9]: I think `WebGLCubeRenderTarget` should be more precise.

Comment (Mugen87) [reply src/scenes/Scene.d.ts:9]: BTW: `import { RenderTarget } from "../renderers/webgl/WebGLRenderLists";` seems incorrect to me. This should be replaced with `WebGLRenderTarget`.

Comment (taphos) [reply src/scenes/Scene.d.ts:8]: done

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**